app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -1,343 +1,320 @@
-* Cassandra 3.11
+This guide walks through downloading, installing, and starting **Kong Enterprise** on **Amazon Linux 2**.
 
-Due to the nature of setting up a Kong cluster with Cassandra, this documentation only covers Postgres. For assistance in setting up Cassandra, please contact your Sales or Support representative.
+The configuration shown in this guide is intended only as an example. Depending on your
+environment, you may need to make modifications and take measures to properly conclude
+the installation and configuration.
 
+Kong supports both PostgreSQL 9.5+ and Cassandra 3.11.* as its datastore. In this guide, we
+show steps to configure PostgreSQL. For assistance in setting up Cassandra, please contact your Sales or Support representative.
 
 ## Prerequisites
 
-To complete this installation guide you will need:
-
-
-* A valid *Bintray* account. You will need your *username*, account *password* and account *API KEY*.
-    * Example:
-        * Bintray Access key = `john-company`
-        * Bintray username = `john-company@kong`
-        * Bintray password = `12345678`
-        * Bintray API KEY = `12234e314356291a2b11058591bba195830`
-* SSH access to a running EC2 instance
-* A valid **Kong Enterprise License** JSON file, this can be found in your Bintray account. See [Accessing Your License](/enterprise/latest/deployment/access-license))
-* HTTPIE with Python3 _(optional)_ - this allows for making easier commands to Kong, see [Step 6. Set up HTTPIE](/#step-6-verfiy-kong-is-receiving-requests) for details.
-
-
-
-## Step 1. Download Kong Enterprise
-
-1. Download the RPM package from Bintray
-  ```
-  $ sudo yum update
-  $ wget 'https://<BINTRAY_USERNAME>:<PASSWORD>@bintray.com/kong/kong-enterprise-edition-aws/rpm' -O bintray-kong-kong-enterprise-edition-aws.repo --auth-no-challenge`
-  ```
-  > Note: this command requires your Bintray account PASSWORD, **not** your API KEY, 
-  > If you receive the error “BAD PORT” ensure your password does not contain any shell meta-characters like $,@, or / 
-
-2. Move the package into `/etc/yum.repos.d/`
-  ```
-  $ sudo mv bintray-kong-kong-enterprise-edition-aws.repo /etc/yum.repos.d/
-  ```
-
-3. Open the package to and add your Bintray credentials to the `baseurl` value (See [Getting your Bintray Credentials] for help)
-  ```
-  $ sudo vi /etc/yum.repos.d/bintray-kong-kong-enterprise-edition-aws.repo
-  ```
-  ```
-  baseurl=https://<BINTRAY_USER>:<BINTRAY_API_KEY>@kong.bintray.com/kong-enterprise-edition-aws
-  ```
-    > Note: Unlike step 1, this command requires your Bintray account API KEY, **not** your PASSWORD.
-
-4. Obtain your Kong Enterprise license file.
-  If you do not already have your license file, you can download it from your account files in Bintray *`https://bintray.com/kong/<YOUR_REPO_NAME>/license#files`*. See [How you Access Your Enterprise License](/enterprise/latest/deployment/access-license) for help.
+To complete this guide you will need:
```

Comment:
Suggest "To complete this installation you will need:", as the user is completing an install action.